.travis.yml

```diff
@@ -17,7 +17,7 @@ cache:
 
 env:
   global:
-  - LIBRDKAFKA_VERSION=v0.11.6
+  - LIBRDKAFKA_VERSION=v1.0.0
```

Comment:
Since [the docs](https://github.com/confluentinc/confluent-kafka-go#installing-librdkafka) mention v1.0.0 or later we could already upgrade to the latest v1.1.0 version which is already released: https://github.com/edenhill/librdkafka/releases/tag/v1.1.0
But not really blocking but it could already contain some bug fixes

Reply:
Yeah true, I didn't want to do that right away, also I'm not sure if that would work, because the go library is still at 1.0.0. We can do this once things are proven to work.